### DISCUSSION TITLE
W5500

### DISCUSSION THREAD

**Discussion by Mbouk** [cesanta/mongoose#2633] [Ask questions, get answers] [answered]
Hi,
can we use Mongoose in UDP with W5500 ?

**Comment by scaprile** [comment, marked as answer]
Did you see any of our examples/tutorials where we say you can't ?

https://github.com/cesanta/mongoose/blob/73a3897e1bf66881297640396909afd0232166e8/examples/device-dashboard/net.c#L67-L69
https://github.com/cesanta/mongoose/tree/master/examples/rp2040/pico-w5500
https://mongoose.ws/documentation/#pico--w5500-module
